packages/flutter_tools/lib/src/aot.dart

```diff
@@ -196,6 +195,7 @@ class AotBuilder {
     String outputDir,
     bool quiet
   }) async {
+    // This code is currently dead, but will be updated as we move iOS to assemble.
```

Comment:
This comment should probably link to a github issue.

Reply:
Maybe also `assert(false)` or `throw 'Unreachable'`.

Reply:
Done